### PR TITLE
fix typos and remove extra semicolons

### PR DESCRIPTION
**Pull request by spacewalkman**
as title

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: src/kvstore/KVStore.h [hunk 1]
```diff
@@ -101,7 +101,7 @@ class KVStore {
                               std::string&& prefix,
                               std::unique_ptr<KVIterator>* iter) = delete;
 
-    // Asynchrous version of remove methods
+    // Asynchronous version of remove methods
     virtual void asyncMultiPut(GraphSpaceID spaceId,
```
Comment (zhangguoqing) [inline src/kvstore/KVStore.h]: The ’asyncMultiPut’ is not the remove methods. Please put the line to 109.

**Comment by spacewalkman** [comment]
Address @zhangguoqing 's comment. Rebased and repushed.

**Comment by dutor** [comment]
Jenkins, go!

**Pull request merged**